bytestring.cabal

```diff
@@ -42,6 +42,7 @@ Description:
 
 License:             BSD3
 License-file:        LICENSE
+Cabal-Version:       1.18
```

Comment:
What's the reason for the increased `Cabal-Version` constraint?

Reply:
Originally I raised it to 2.* because I needed `elif`. I guess I changed it back to too high a number.

Reply:
In this case, please revert to the original setting.

Reply:
@kozross could you please revert to `Cabal-Version:       >= 1.10`? I'd like to merge the branch.